data/json/monstergroups/bugs.json

```
@@ -84,10 +84,171 @@
+        "conditions": [ "SPRING", "SUMMER", "AUTUMN" ]
+      },
+      {
+        "monster": "mon_wasp_guard",
+        "freq": 500,
+        "cost_multiplier": 1,
+        "starts": 540,
+        "pack_size": [ 2, 4 ],
+        "conditions": [ "SPRING", "SUMMER", "AUTUMN" ]
+      }
+    ]
+  },
+  {
+    "name": "GROUP_WASP_FORAGER",
+    "type": "monstergroup",
+    "default": "mon_null",
+    "monsters": [
+      {
+        "monster": "mon_wasp_small",
+        "freq": 1000,
+        "cost_multiplier": 1,
+        "ends": 180,
+        "conditions": [ "SPRING", "SUMMER", "AUTUMN" ]
+      },
+      {
+        "monster": "mon_wasp_small",
+        "freq": 500,
+        "cost_multiplier": 1,
+        "starts": 180,
+        "ends": 540,
+        "pack_size": [ 2, 4 ],
+        "conditions": [ "SPRING", "SUMMER", "AUTUMN" ]
+      },
+      {
+        "monster": "mon_wasp",
+        "freq": 1000,
+        "cost_multiplier": 1,
+        "starts": 360,
+        "ends": 720,
+        "conditions": [ "SPRING", "SUMMER", "AUTUMN" ]
+      },
+      {
+        "monster": "mon_wasp",
+        "freq": 1000,
+        "cost_multiplier": 1,
+        "starts": 720,
+        "pack_size": [ 2, 4 ],
+        "conditions": [ "SPRING", "SUMMER", "AUTUMN" ]
+      }
+    ]
+  },
+  {
+    "name": "GROUP_WASP_GUARD",
+    "type": "monstergroup",
+    "default": "mon_null",
+    "monsters": [
+      {
+        "monster": "mon_wasp_small_guard",
+        "freq": 1000,
+        "cost_multiplier": 1,
+        "ends": 180,
+        "conditions": [ "SPRING", "SUMMER", "AUTUMN" ]
+      },
+      {
+        "monster": "mon_wasp_small_guard",
+        "freq": 500,
+        "cost_multiplier": 1,
+        "starts": 180,
+        "ends": 540,
+        "pack_size": [ 2, 4 ],
+        "conditions": [ "SPRING", "SUMMER", "AUTUMN" ]
+      },
+      {
+        "monster": "mon_wasp_guard",
+        "freq": 500,
+        "cost_multiplier": 1,
+        "starts": 180,
+        "ends": 540,
+        "conditions": [ "SPRING", "SUMMER", "AUTUMN" ]
+      },
+      {
+        "monster": "mon_wasp_guard",
+        "freq": 500,
+        "cost_multiplier": 1,
+        "starts": 540,
+        "pack_size": [ 2, 4 ],
+        "conditions": [ "SPRING", "SUMMER", "AUTUMN" ]
+      }
+    ]
+  },
+  {
+    "name": "GROUP_WASP_QUEEN",
+    "type": "monstergroup",
+    "default": "mon_null",
+    "monsters": [
+      { "monster": "mon_wasp_queen", "freq": 1000, "cost_multiplier": 50, "ends": 720 },
+      { "monster": "mon_wasp_mega", "freq": 1000, "cost_multiplier": 50, "starts": 360 }
+    ]
+  },
+  {
+    "name": "GROUP_WASP_PUPA",
     "type": "monstergroup",
     "default": "mon_wasp",
-    "monsters": [ { "monster": "mon_wasp", "freq": 100, "cost_multiplier": 1, "conditions": [ "SPRING", "SUMMER", "AUTUMN" ] } ]
+    "monsters": [
+      { "monster": "mon_dermatik", "freq": 50, "cost_multiplier": 1 },
+      { "monster": "mon_wasp_guard", "freq": 500, "cost_multiplier": 1 }
```

I assume one of these was meant to be pupa given the ID?
```suggestion
      { "monster": "mon_wasp_guard", "freq": 50, "cost_multiplier": 1 },
      { "monster": "mon_wasp_pupa", "freq": 500, "cost_multiplier": 1 }
```